### ISSUE TITLE
Accessibility: Improving Acessibility of python.org/documentation.html

### ISSUE THREAD

**Issue by Biki-das** [jupyterhub/outreachy#111]
### What page is this for?

https://ipython.org/documentation.html

### WAVE accessibility report

![Documentation — IPython — Mozilla Firefox 10_21_2022 10_48_30 PM](https://user-images.githubusercontent.com/72331432/197253253-b7737015-8a86-4d4c-810d-ad507ce24397.png)
![Documentation — IPython — Mozilla Firefox 10_21_2022 10_48_38 PM](https://user-images.githubusercontent.com/72331432/197253318-a2feecb3-00bd-4442-9ff6-2e872b193566.png)
![Documentation — IPython — Mozilla Firefox 10_21_2022 10_48_49 PM](https://user-images.githubusercontent.com/72331432/197253345-42f9fb3d-3152-4e1c-871f-dafa532bec25.png)

### The HTML element to be changed

* The html element is missing the Lang attribute which violates the accessibility practices
```
<html xmlns="http://www.w3.org/1999/xhtml" style="--pulse-color: #fffc0230; margin-left: 380px;">
```
* The color of the anchor tag has color contrast issues.
```
<a href="https://jupyter.readthedocs.io/en/latest/install.html" target="_blank">Install</a>
```
*  The div has a css property of text-align:justify which makes it less accessible
```
<div class="body" role="main" style=""></div>
```
```
* The img tag alt attributes should have more explicit information about the image
<img class="logo" src="_static/IPy_header.png" alt="Logo">
```
* the img tag alt needs an alt attribute to make it accessible to screen readers
```
<img src="_static/nbviewer-thm-2.png" title="IPython Notebook Viewer" style="">
```
* the img tag alt needs an alt attribute to make it accessible to screen readers
```
<img title="Packt Publishing donates a portion of the proceeds from this book to
support IPython's development." id="sidebar-book-cover" style="margin-bottom: 10px;" src="_static/mastering-ipython-book.png" width="160" height="200">
```
* The a tag links to an external pdf which has to either made accessible natively or inform the user that the link will open a PDF document
```
<a class="reference external" href="https://ipython.org/ipython-doc/rel-0.9/ipython.pdf">PDF</a>
```

### Your proposed HTML (or CSS), after change

* Inserting the `lang` attribute solve the language invalid error 

```
<html  lang="en" xmlns="http://www.w3.org/1999/xhtml" style="--pulse-color: #fffc0230; margin-left: 380px;">
```
* Adjusting the Foreground text color helps us to achieve the accessible color contrast 
![Developer Tools — Documentation — IPython — https___ipython org_documentation html# 10_21_2022 11_49_48 PM](https://user-images.githubusercontent.com/72331432/197263513-65f02d4e-94cb-464c-a6e6-88fde960714f.png)
![Documentation — IPython — Mozilla Firefox 10_21_2022 11_48_37 PM](https://user-images.githubusercontent.com/72331432/197263694-7dfd6183-34a3-4699-891d-80e7a8efea61.png)

* Making the `alt` attribute more explicit for the image makes it more accessible and the screen readers will read the information aloud and concise

```
<img class="logo" src="_static/IPy_header.png" alt="HeaderLogo">
```
* Inserting an `alt` attribute in the image will allow screen readers to pick the image info

```
<img src="_static/nbviewer-thm-2.png" title="IPython Notebook Viewer" alt="IPython Notebook Viewer""style="">
```

* Inserting an `alt` attribute in the image will allow screen readers to pick the image info
```
<img title="Packt Publishing donates a portion of the proceeds from this book to
support IPython's development." alt="Book" id="sidebar-book-cover" style="margin-bottom: 10px;" src="_static/mastering-ipython-book.png" width="160" height="200" >
```
* Linking to any external pdf document makes it less accessible, though it is still not a major violation of accessibility practices, we can try to make the pdf natively accessible

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
